test/schema-feature.test.js

```diff
@@ -170,7 +170,7 @@ test('Cannot add schema for query and querystring', t => {
 
 test('Should throw of the schema does not exists in input', t => {
   t.plan(2)
-  const fastify = Fastify()
+  const fastify = Fastify({ exposeHeadRoutes: false })
```

Comment:
what happens if this is set to true?

Reply:
The error validates a throw on an unexisting schema reference, but the test is waiting for the error to be thrown on the original `GET /:id` endpoint. But because of the order of the routes being loaded (first `GET` , second `HEAD`), the validation always fails on the `HEAD` route throwing exactly the same error but just for the sibling route

Reply:
Can you add a test with `exposeHeadRoutes: true`?